modules/client/examples/04-addresslistVoting-client/01-installation.ts

```diff
@@ -1,6 +1,6 @@
 /* MARKDOWN
```

Comment:
No other files changes for Addresslist voting? Token voting and multisig have plenty of renames